vignettes/retrieved_dropout.Rmd

```diff
@@ -0,0 +1,318 @@
+---
+title: "Implementation of retrieved-dropout models using rbmi"
+author: "Alessandro Noci, Marcel Wolbers"
+date: "2024-07-30"
+output: 
+  bookdown::html_document2:
+    toc: true
+    toc_depth: 4
+    number_sections: true
+    citation_package: natbib
+    base_format: rmarkdown::html_vignette
+bibliography: "references.bib"
+link-citations: true   
+linkcolor: blue
+pkgdown:
+  as_is: true
+vignette: >
+  %\VignetteIndexEntry{Implementation of retrieved-dropout models using rbmi}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+This vignette describes how retrieved dropout models which include time-varying intercurrent event (ICE) indicators can be 
+implemented in the `rbmi` package.
+
+# Retrieved dropout models in a nutshell
+
+Retrieved dropout models have been proposed for the analysis of estimands using the treatment policy strategy for addressing an ICE. 
+In these models, missing outcomes are multiply imputed conditional upon whether they occur pre- or post-ICE. Retrieved dropout models
+typically rely on an extended missing-at-random (MAR) assumption, i.e., they assume that missing outcome data is similar to observed 
+data from subjects in the same treatment group with the same observed outcome history, and the same ICE status. For a more 
+comprehensive description and evaluation of retrieved dropout models, we refer to @Guizzaro2021, @PolverejanDragalin2020, @Noci2021, 
+@Drury2023, and @Bell2024. Broadly, these publications find that retrieved dropout models reduce bias compared to alternative 
+analysis approaches based on imputation under a basic MAR assumption or a reference-based missing data assumption. However, several 
+issues of retrieved dropout models have also been highlighted. Retrieved dropout models require that enough post-ICE data is 
+collected to inform the imputation model. Even with relatively small amounts of missingness, complex retrieved dropout models may 
+face identifiability issues. In addition, retrieved dropout models have been associated with inflated standard errors of associated 
+treatment effect estimators which has a detrimental effect on study power. In particular, it has been observed that once the post-ICE
+observation percentage falls below 50%, the power loss can be quite dramatic [@Bell2024]. 
```

Comment:
I don't think it refers to a "jump" in the power loss. We want to say that particular care is needed with power loss, if the proportion of subjects with observed post-ICE data is low. I guess it will be somehow a non-linear behavior where the power decreases by little if the post-ICE observed data is high and then it gets "steeper" with decreasing post-ICE observed data. 50% is just a threshold used. Do you think it needs re-phrasing?

Reply:
I looked at this again and another issue is that the sentence is not particularly precise. E.g. the bell2024 citation talks about power loss relative to ref-based methods but we just say "power loss". 

My suggestion is to remove the sentence altogether and add the bell2024 reference to the end of the previous sentence.
To me, this is cleaner as the purpose of the vignette is to demonstrate how these methods can be implemented and not to provide advice when should be used. For deciding the latter, the user will have to read bell2024 and other references in any case (and perhaps make their own simulations).

@nociale and @gowerc, what do you think?

PS: For reference, this is the relevant sentence from bell2024: 
> A drawback to these models in general is the loss of power relative to the best performing RBI model (J2R), which
becomes meaningful for post-IE observation percentages below 50% and increases at an accelerating rate as this
percentage decreases.

Reply:
Ok!